### PR TITLE
#2477 Python Cheat Sheet:Added some Dictionary Methods and Array Methods

### PR DESCRIPTION
**Pull request by shubhamrajput**
Added one section of Dictionary Methods and one Array method to Python Cheat sheet
IA Page: https://duck.co/ia/view/python_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@shubhamrajput Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
## [Python Language Cheat Sheet](https://duck.co/ia/view/python_cheat_sheet)

**Description**: Overview of common Python methods

**Example Query**: [python cheat sheet](https://beta.duckduckgo.com/?q=python cheat sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/python_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/python_cheat_sheet)*

File: share/goodie/cheat_sheets/json/apt-get.json [hunk 1]
```diff
@@ -23,10 +23,13 @@
                 "key": "apt-get install <package_name>"
             },
             {
+<<<<<<< HEAD
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/apt-get.json]: This is the problem @shubhamrajput

**Comment by mintsoft** [comment]
Thanks @shubhamrajput

**Comment by moollaza** [comment]
@shubhamrajput thanks again for improving the Python Cheat Sheet! You’ve done a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Pull request merged**